to_do/src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import './App.css';
+import { ToastContainer, toast } from 'react-toastify';
```

Comment:
Forgot to gitignore the toast component.

Keep in mind that every time you use a npm module, you want to ignore it on the gitignore file